test/index.test.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const Knex      = require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  },
+  useNullAsDefault: true
```

Comment:
Sorry that was rather opaque. When using Knex with sqlite, you have to explicitly specify if you want to replace undefined keys with NULL during an insert or not. Otherwise you get the warning message above every time you run a file that initializes Knex.